training/training/routes/image/image.py

```diff
@@ -1,40 +1,18 @@
-from typing import Literal, Optional
-from django.http import HttpRequest
-from ninja import Router, Schema
-from training.core.criterion import getCriterionHandler
-from training.core.dl_model import DLModel
-from training.core.dataset import ImageDefaultDatasetCreator
-from torch.utils.data import DataLoader
-from training.core.optimizer import getOptimizer
-from training.core.trainer import ClassificationTrainer
+from ninja import Router
 from training.routes.image.schemas import ImageParams
-from training.core.authenticator import FirebaseAuth
+from training.core.authenticator import FirebaseAuth, Request
+from training.celery_app import celery_app
+
+import uuid
 
 router = Router()
 
 
 @router.post("", auth=FirebaseAuth())
-def imageTrain(request: HttpRequest, imageParams: ImageParams):
-    if imageParams.default:
-        dataCreator = ImageDefaultDatasetCreator.fromDefault(imageParams.default)
-        train_loader = dataCreator.createTrainDataset()
-        test_loader = dataCreator.createTestDataset()
-        model = DLModel.fromLayerParamsList(imageParams.user_arch)
-        optimizer = getOptimizer(model, imageParams.optimizer_name, 0.05)
-        criterionHandler = getCriterionHandler(imageParams.criterion)
-        if imageParams.problem_type == "CLASSIFICATION":
-            trainer = ClassificationTrainer(
-                train_loader,
-                test_loader,
-                model,
-                optimizer,
-                criterionHandler,
-                imageParams.epochs,
-                dataCreator.getCategoryList(),
-            )
-            for epoch_result in trainer:
-                print(epoch_result)
-            print(trainer.labels_last_epoch, trainer.y_pred_last_epoch)
-            print(trainer.generate_confusion_matrix())
-            print(trainer.generate_AUC_ROC_CURVE())
-            return trainer.generate_AUC_ROC_CURVE()
+def imageTrain(request: Request, imageParams: ImageParams):
```

Comment:
very cool that the django endpoint is simpler to follow since this the heavy lifting will be done with celery executing the training requests from sqs

Reply:
yea, that's correct